### PR TITLE
Add guide for Azure Postgres/MySQL database access

### PR DESCRIPTION
**Pull request by r0mant**
This PR adds a new database access guide page that explains how to setup database access with Azure-hosted Postgres or MySQL databases. The actual functionality was merged in master (https://github.com/gravitational/teleport/pull/8951) and v8 (https://github.com/gravitational/teleport/pull/9185) a few weeks ago so it's already supported in the product, just the guide has been missing.

### CODE REVIEW

**Comment by r0mant** [comment]
@klizhentas @ptgott @xinding33 Could you guys review this please? I'd like to deploy this sooner than later so people can start using database access with Azure Postgres/MySQL. Thanks!

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:151]:
```suggestion
  Teleport Database Service uses Azure SDK's default credential provider chain to
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:154]:
```suggestion
  environment-based authentication with a client secret, Database Service should
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:172]:
```suggestion
Only the Azure AD administrator for the database can connect to it and create
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:173]:
```suggestion
Azure AD users. Go to your database's _Active Directory admin_ page
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:174]:
```suggestion
and set the AD admin using the _Set admin_ button:
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:180]:
```suggestion
### Connect to the database as an AD admin
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:184]:
```suggestion
Use the Azure `az` CLI utility to log in as the user that you set as the AD admin,
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:210]:
```suggestion
service principal that Teleport Database Service will be using. Use `Client ID`
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:236]:
```suggestion
Log into your Teleport cluster. Your Azure database should appear in the list of
```

File: docs/pages/database-access/guides/azure-postgres-mysql.mdx
Comment (ptgott) [inline docs/pages/database-access/guides/azure-postgres-mysql.mdx:265]: Does this refer to the end-user's machine or the VM running the Database Service?

**Comment by r0mant** [comment]
@ptgott Thank you for the review! I have addressed your feedback, could you take another look?

**Review by ptgott — APPROVED** [review]

**Comment by r0mant** [comment]
@klizhentas @xinding33 Need a second review on this from either of you. Could you take a look please?

**Review by xinding33 — APPROVED** [review]

**Pull request merged**
